src/zenml/zen_stores/migrations/env.py

```diff
@@ -37,7 +37,7 @@ def run_migrations() -> None:
     if store_cfg is None or store_cfg.type != StoreType.SQL:
         store_cfg = GlobalConfiguration().get_default_store()
 
-    store = SqlZenStore(
+    store = SqlZenStore(  # type: ignore
```

Comment:
ignore what ? best to be more specific about what fails here

Reply:
don't remember writing this at all :D